src/core_modules/capture-core/components/Widgets/ProfileWidget/profileWidget.types.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+export type Props = {|
+    classes: {
+        flatListWrapper: string;
+        itemRow: string;
+        itemKey: string;
+    }
```

Comment:
Thats not by any means bad annotation here, but you might wanna consider using `CssClasses` type we have "globally" defined. This way you dont have to all _every single_ class in this annotation here. I used to find it a bit annoying that flow will complain because a single class was missing from the annotation.

You can find that global definition in the file `src/core_modules/capture-core/flow/typeDeclarations.js` if you wanna check it out

Reply:
Nothing wrong with the approach here @huocha, but as @paschalidi says we've been using the more pragmatic `CssClasses` lately because it gets cumbersome to annotate the classes. I would say this one is up to you, nothing wrong with keeping this as is.